src/main/java/org/yomigae/output/StreamingACNDatagram.java

```diff
@@ -86,7 +88,7 @@ protected StreamingACNDatagram(int universeNumber, int dataSize) {
   }
 
   public StreamingACNDatagram(int universeNumber, DmxFragment fragment) {
-		this(universeNumber, fragment.getNumChannels());
+		this(universeNumber, DATA_PAYLOAD_SIZE);
```

Comment:
Does `fragment.getNumChannels()` not work here?

Reply:
It worked, except we ran into a weird bug where the last light in the chain wasn't responding.

Tracy had mentioned separately that he has run into strange behaviors with certain fixtures in the past and that sending full packets of 512 bytes fixed them.  I tried that, and it worked.

I'm glad he mentioned it--I feel like I would never have thought to try this.